packages/backend/src/config/parseConfig.ts

```diff
@@ -285,6 +286,7 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
                 ) || 5000,
         },
         s3: {
+            region: process.env.S3_REGION,
```

Comment:
what happens if S3_REGION is undefined ?

Reply:
Doesn't work, throws an error saying region is missing.